CHANGELOG.md

```diff
@@ -37,6 +37,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Aggregate ignores frames that crosses aggregate boundary (#248)
 - The `report` and `aggregate` commands with `--json` option now correctly
   encode Arrow objects (#329)
+- Make after-edit-check ensure that edited time stamps are not in the future
+  (#381)
```

Comment:
Should be in the `fixed` subsection of the `Unreleased` section.

Reply:
Apologies. Must have been too tired. Should be fixed now.